src/main/java/com/example/controller/BookController.java

```diff
@@ -29,24 +30,28 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
+    @PreAuthorize("hasRole('USER')")
```

Comment:
In such way you give no access to endpoint to Admin, please recheck such behaviour is intended

Reply:
Its also strange for me, but according to requirements here shouldn't be get method accessable:
![image](https://github.com/maiiat/online-book-store/assets/43042799/bf22eed3-fd28-4bd0-904e-d4cfcb3c1a39)